db/db_setup.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 port=${1:-5432} # can override port from command line
 PGPASSWORD=postgres psql -U postgres -h localhost -p $port -f create_db.sql
-migrate -database "postgres://postgres:postgres@localhost:$port/cms?sslmode=disable" -path migrations up
+migrate -database "postgres://ourroots_schema:password@localhost:$port/cms?sslmode=disable" -path migrations up
```

Comment:
just curious, why a separate username for the migration?

Reply:
So the normal app user doesn't deed permission to drop tables. `ourroots_schema` is set up as the equivalent of the `postgres` user in a normal install.

Reply:
ok, that makes sense.